config/settings_data.json

```diff
@@ -1,45 +1,157 @@
 {
-    "current": "Gray",
+    "current": "Cyan",
     "presets": {
         "Blue": {
             "color_primary": "#007bff",
             "color_secondary": "#ff8400",
-            "link_color": "#fff"
+            "link_color": "#fff",
+            "footer_quicklinks_enable": true,
+            "footer_quicklinks_linklist": "footer",
+            "footer_content_enable": true,
+            "footer_social_enable": true,
+            "social_twitter_link": "https://twitter.com/virtocommerce",
+            "social_facebook_link": "https://facebook.com/virtocommerce",
+            "social_pinterest_link": "",
+            "social_google_plus_link": "https://plus.google.com/+virtocommerce",
+            "social_instagram_link": "https://instagram.com/virtocommerce",
+            "social_tumblr_link": "",
+            "social_youtube_link": "",
+            "social_vimeo_link": "",
+            "social_fancy_link": "",
+            "footer_legallinks_enable": true
```

Comment:
Move this settings to place under `"current"`

Reply:
@asvishnyakov These settings don't work outside presets